tests/components/SingleAddon.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import SingleAddon from '../../app/js/components/manageApps/SingleAddon';
+
+
+describe('<SingleAddon/>', () => {
+
+  const app =
+    {"appDetails":
+      {
+        "version":"0.1.0",
+        "name":"openmrs-addOnManager",
+        "description":"This will help in uploading the openmrs open web apps and omod files",
+        "icons":
+          {
+            "16":null,
+            "48":"/img/omrs-button.png",
+            "128":null,
+          },
+        "developer":
+          {
+            "url":"https://github.com/openmrs/openmrs-owa-addonmanager",
+            "name":"andela-pupendo,kingisaac95, andela-wanjikum, andela-jomadoye",
+            "company":null,
+            "email":null,
+          },
+        "activities":
+          {
+            "openmrs":
+              {
+                "href":"*",
+              }
+          },
+        "folderName":"openmrs-addonmanager",
+        "baseUrl":null,
+        "launchUrl":null,
+        "launch_path":"index.html",
+        "installs_allowed_from":null,
+        "default_locale":"en",
+        "deployed.owa.name":null,
+      }
+      ,"appType":"owa",
+      "install":false,
+    }
+
+  const addonComponent =
+    mount(<SingleAddon
+            app={app}
+            addonParam={() => {}}
+            updatesVersion={{}} />);
+
+  it('Should render', () => {
```

Comment:
Make the assertion messages more descriptive e.g `should render <SingleAddon  />`